browser/ui/views/frame/vertical_tab_strip_widget_delegate_view.cc

```diff
@@ -18,8 +18,81 @@
 
 #if defined(USE_AURA)
 #include "ui/views/view_constants_aura.h"
+#include "ui/views/view_utils.h"
+#else
+#include "chrome/browser/ui/browser_list.h"
```

Comment:
nit: As both headers are not specific to aura, both could be included w/o aura condition.

Reply:
I see where you're coming from! Done.